controllers/volumereplication_controller.go

```diff
@@ -64,3 +86,34 @@ func (r *VolumeReplicationReconciler) SetupWithManager(mgr ctrl.Manager, cfg *co
 		For(&replicationv1alpha1.VolumeReplication{}).
 		Complete(r)
 }
+
+func (r VolumeReplicationReconciler) getVolumeHandle(instance *replicationv1alpha1.VolumeReplication) (string, error) {
```

Comment:
can we move this to a new file `pvc.go` ?

Reply:
pkg/pvc/pvc.go   or controllers/pvc.go ?

Reply:
it should be in controllers/pvc.go as you are using r.Client to get the objects?